### ISSUE TITLE
Implement all Request and Response methods to query data from raw objects

### ISSUE THREAD

**Issue by ggalmazor** [ggalmazor/jfry#3]
Like in http://sparkjava.com/documentation.html#request and http://sparkjava.com/documentation.html#response

**Comment by ribomation** [comment]
Agree.
When it comes to reading the raw request, I would appreciate to see a use case of a file upload operation.

**Comment by ggalmazor** [comment]
Yes, this is something that has high priority on the roadmap but I haven't thought any design for it yet... Suggestions are more than welcome :)
